src/views/Player.vue

```diff
@@ -258,11 +297,37 @@ export default {
 
         // initializing the retention array with zeros
         this.retention = Array(this.player.duration).fill(0);
+        this.submitted = Array(this.ivideo_questions.length).fill(0);
+        this.skipped = Array(this.ivideo_questions.length).fill(0);
+        this.revised = Array(this.ivideo_questions.length).fill(0);
+
+        const play_buttons = document.querySelectorAll("[data-plyr='play']")
+        play_buttons[0].addEventListener(
+          'click', this.listenToPlayButtons, false
+        );
+
+        play_buttons[1].addEventListener(
+          'click', this.listenToPlayButtons, false
+        );
+
       });
 
       player.on('play', event => {
         const instance = event.detail.plyr;
         instance.fullscreen.enter()
+
+        if (this.player.currentTime < 10 && this.player.playing==true){
+          setTimeout(() => {
+            console.log(this.player.currentTime)
+            if (this.player.currentTime == 0 && this.player.playing==true) {
+              // browser unsupported -> show error page
+              this.isSeekBarMoving = false
+              this.isBrowserSupported = false
+            }
+            this.uploadJson()
+          }, 5000);
+        }
+
```

Comment:
ye thoda detail mei explain kar de yaar kya ho raha yahan par and console.log hata de 
and ==true nahi use karna chaiye - ideally buss if (variable) {} should work
also, 10 aur 5000 ko kuch variable banake store kar de upar - warna hardcoded hard to find

Reply:
Iska oopar likha tha ye samajh aya ya aur explain karna hai?

"This is the piece of code which will help us detect/handle the cases where the video does not play (stupid browser case).

We need to do this only in the first 5 seconds. If the video is playing, then we'll wait for about 5 seconds and see if the seek bar is still at time 0. If that's the case, then isSeekBarMoving is set to false and isBrowserSupported is set to false, and the user sees an error page."

Reply:
1. Is there any case where `isSeekBarMoving` and `isBrowserSupported` have different values? if not, then they can be replaced with a single variable.

2. Why not set this `setTimeOut` only at the beginning of playback? If the player starts playing, wont' this set timeout again and again many times?

Reply:
1. No, right now they both have same values, as browser being supported is dependent on if seek bar is moving. Yes we can keep it as one variable. `IsBrowserSupported` ka support pehle se tha where it used to look up from the array of supported browsers. Abhi agar vo functionality nahi rakhni hai toh we can combine both these variables

2. Beginning of the playback ke lie hi I've put `this.player.currentTime < 10` 
Is there any other way to do that?

Reply:
2 ke liye Chalega I guess. I was wrong. It will be multiple listeners only if user stops and starts playback multiple times. So is okay!